third_party/terraform/resources/resource_iam_binding.go.erb

```diff
@@ -62,8 +89,11 @@ func resourceIamBindingCreateUpdate(newUpdaterFunc newResourceIamUpdaterFunc, en
 
 		binding := getResourceIamBinding(d)
 		modifyF := func(ep *cloudresourcemanager.Policy) error {
-			cleaned := removeAllBindingsWithRole(ep.Bindings, binding.Role)
+			cleaned := removeAllBindingsWithRoleAndCondition(ep.Bindings, binding.Role, binding.Condition)
```

Comment:
While you're here, can you rename this to something other than remove? Every time I read it, I think it's going to remove something from the API. I'm partial to `filterBindingsWithRoleAndCondition`, personally.

Reply:
Done